src/script/page/MainContent/panels/preferences/devices/components/DetailedDevice.tsx

```diff
@@ -39,7 +39,9 @@ const DetailedDevice: React.FC<DeviceProps> = ({device, fingerprint}) => {
       </h3>
 
       <p className="preferences-devices-id">
-        <span>{t('preferencesDevicesId')}</span>
+        <span>
+          <strong>{t('preferencesDevicesId')}</strong>
+        </span>
```

Comment:
We need this 2 tags ?
```suggestion
      <strong>{t('preferencesDevicesId')}</strong>
```